package.json

```diff
@@ -38,7 +38,7 @@
   "scripts": {
     "start": "start-storybook",
     "build": "build-storybook -c .storybook -o public",
-    "test": "react-scripts test",
+    "test": "cp tsconfig.json tsconfig.json.bak && react-scripts test && cp tsconfig.json.bak tsconfig.json && rm tsconfig.json.bak",
```

Comment:
Optionally use ‘mv’ instead of ‘cp’ and ‘rm’ after the test run

Reply:
oh, good call. that saves a command.